Doctrine/PHPCR/ElasticaToModelTransformer.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace FOS\ElasticaBundle\Doctrine\PHPCR;
+
+use FOS\ElasticaBundle\Doctrine\AbstractElasticaToModelTransformer;
+
+/**
+ * Maps Elastica documents with Doctrine objects
+ * This mapper assumes an exact match between
+ * elastica documents ids and doctrine object ids
+ */
+class ElasticaToModelTransformer extends AbstractElasticaToModelTransformer
+{
+    /**
+     * Fetch objects for theses identifier values
+     *
+     * @param array $identifierValues ids values
+     * @param Boolean $hydrate whether or not to hydrate the objects, false returns arrays
+     * @return array of objects or arrays
+     */
+    protected function findByIdentifiers(array $identifierValues, $hydrate)
+    {
+        // Special case where model is interface
+        $reflectionClass = new \ReflectionClass($this->objectClass);
+        if ($reflectionClass->isInterface()) {
```

Comment:
i think [interface_exists](http://php.net/manual/en/function.interface-exists.php) is exactly what you want here.